src/main/kotlin/controller/BlackJackController.kt

```diff
@@ -0,0 +1,62 @@
+package controller
+
+import model.Dealer
+import model.Player
+import service.ResultCalculator
+import util.isYes
+import view.InputView
+import view.OutputView
+
+object BlackJackController {
+    fun playGame() {
+        val players = getPlayerNames().map { Player(it) }
+        val dealer = Dealer()
+        startGame(players, dealer)
+        runTurns(players, dealer)
+        showResults(players, dealer)
+    }
+
+    private fun getPlayerNames(): List<String> {
+        val players = InputView.requestPlayerNames()
+        require(players.size <= 6) { "Maximum player names must be 6" }
+        return players
+    }
+
+    private fun startGame(
+        players: List<Player>,
+        dealer: Dealer,
+    ) {
+        dealer.giveInitialCardsToPlayers(players)
+        dealer.drawInitialCards()
+        OutputView.displayInitialCards(players, dealer)
```

Comment:
Nice!
lot of the business logic that was exposed in the controller is now nicely hidden. 👍